packages/hash/api/src/model/knowledge/hashInstance.model.ts

```diff
@@ -115,18 +118,47 @@ export default class extends EntityModel {
   ): Promise<boolean> {
     const { userModel } = params;
 
-    return await LinkModel.get(graphApi, {
-      sourceEntityId: this.entityId,
-      linkTypeId: SYSTEM_TYPES.linkType.admin.schema.$id,
-      targetEntityId: userModel.entityId,
-    })
-      .then(() => true)
-      .catch((error: Error) => {
-        if (error instanceof NotFoundError) {
-          return false;
-        }
-        throw error;
-      });
+    const outgoingAdminLinkEntityModels = await LinkEntityModel.getByQuery(
+      graphApi,
+      {
+        all: [
+          {
+            equal: [
+              { path: ["leftEntity", "uuid"] },
+              { parameter: this.entityUuid },
+            ],
```

Comment:
Wherever we do 'equal uuid' and we also do 'equal ownedById` please? :)

We intend to change structural queries (probably) to only accept an `entityId` anyway in the long run, but for now lets try and be explicit and correct just in case.

Reply:
Addressed in https://github.com/hashintel/hash/pull/1435/commits/8aa3152488cf511718d875ba0dfa8cbfcdf30d2d